### PR TITLE
Shorten "geometry locked" and "read only" texts on layer properties

### PR DESCRIPTION
**Pull request by m-kuhn**
Peek 2020-12-13 13-27

### CODE REVIEW

**Review by suricactus — APPROVED** [review]

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 5de0c8e9d96bd89f9dfade5d441a7dc696a93a34:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1478-55b694a016e6674f7434f63d1fc94c0b05c26fab-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1478-55b694a016e6674f7434f63d1fc94c0b05c26fab-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1478-55b694a016e6674f7434f63d1fc94c0b05c26fab-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1478-55b694a016e6674f7434f63d1fc94c0b05c26fab-x86.apk)

**Comment by m-kuhn** [comment]
Another idea worth exploring: should this be moved to the bottom. It's something you need to read once only, so it should not be the topmost most prominent thing.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 44d0d3aa178f8956cb5d312ca6027ccded044a81:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1478-cb26841024a55f8f82bffa6c5a9c3cb0a3131b34-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1478-cb26841024a55f8f82bffa6c5a9c3cb0a3131b34-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1478-cb26841024a55f8f82bffa6c5a9c3cb0a3131b34-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1478-cb26841024a55f8f82bffa6c5a9c3cb0a3131b34-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 4818b429280eb3e1bfa412df55c2144acb4b12fd:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1478-c92fc75a0a3c352c93a903129fbe1695c4138142-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1478-c92fc75a0a3c352c93a903129fbe1695c4138142-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1478-c92fc75a0a3c352c93a903129fbe1695c4138142-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1478-c92fc75a0a3c352c93a903129fbe1695c4138142-x86.apk)

File: src/qml/LayerTreeItemProperties.qml [hunk 1]
```diff
@@ -196,6 +178,34 @@ Popup {
...
+              + (isReadOnly ? qsTr('Read-Only Layer') : qsTr('Geometry Locked Layer'))
+        font: Theme.tipFont
+
+        MouseArea {
+            anchors.fill: parent
+            onClicked: {
+                if ( lockText.isReadOnly )
+                    displayToast(qsTr('This layer is configured as "Read-Only" which disables adding, deleting and editing features.'))
+                else
+                    displayToast(qsTr('This layer is configured as "Lock Geometries" which disables adding and deleting features, as well as modifying the geometries of existing features.'))
```
Comment (suricactus) [inline src/qml/LayerTreeItemProperties.qml:205]:
```suggestion
                if ( lockText.isReadOnly ) {
                    displayToast(qsTr('This layer is configured as "Read-Only" which disables adding, deleting and editing features.'))
                } else {
                    displayToast(qsTr('This layer is configured as "Lock Geometries" which disables adding and deleting features, as well as modifying the geometries of existing features.'))
                }
```
I would either do it like this, or with `? :` operator.

Comment (suricactus) [reply src/qml/LayerTreeItemProperties.qml:205]: too late

Comment (m-kuhn) [reply src/qml/LayerTreeItemProperties.qml:205]: sorry, I was too late

**Pull request merged**